# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mdnsresponder/linkflags.patch.cross
# Copyright (C) 2009 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/mDNSPosix/Makefile
+++ b/mDNSPosix/Makefile
@@ -322,7 +322,7 @@ $(INSTBASE)/sbin/mdnsd: $(BUILDDIR)/mdns
 
 $(INSTBASE)/lib/libdns_sd.$(LDSUFFIX).$(LIBVERS): $(BUILDDIR)/libdns_sd.$(LDSUFFIX)
 	$(CP) $< $@
-	$(LN) $@ $(INSTBASE)/lib/libdns_sd.$(LDSUFFIX)
+	$(LN) libdns_sd.$(LDSUFFIX).$(LIBVERS) $(INSTBASE)/lib/libdns_sd.$(LDSUFFIX)
 ifdef LDCONFIG
 	# -m means 'merge into existing database', -R means 'rescan directories'
 	$(LDCONFIG) -mR
